event-processing/event-processing-template.yml

```diff
@@ -3602,3 +3608,90 @@ Resources:
 
 
   # TXMA2 QueryResults resources end
+
```

Comment:
i noticed all your resources are under 

```
# TXMA2 QueryResults resources end
```

about 300 lines away from the rest of the publish to accounts resources, can you place all resources related to publishing to accounts together

Reply:
Done